XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -313,4 +477,171 @@ public Bar(double length, int max)
         public double Diameter { get; set; }
         public double MaxDiameter { get; set; }
     }
+
+    public class Recipe
+    {
+        public int Num { get; set; }
+        public string Ingredients { get; set; }
+        public List<string> IngList { get; set; }
+        public string Name { get; set; }
+        public string Color { get; set; }
+        public int numIngredients {

```

Comment:
>numIngredients [](start = 19, length = 14)

capitalize N. Is this needed since it is just count of IngList ?

Reply:
removed this